.travis.yml

```diff
@@ -1,20 +1,44 @@
+os: linux
+dist: trusty
+sudo: false
+group: edge
+
 language: python
 python:
-  - "2.6"
-  - "2.7"
-  - "3.3"
-  - "3.4"
-  - "3.5"
-  - "3.6"
+- 2.6
+- 2.7
+- 3.3
+- 3.4
+- 3.5
+- &mainstream_python 3.6
 
 install:
-  - "travis_retry pip install nose"
-  - "travis_retry pip install coveralls"
-  - "travis_retry pip install webassets"
-  - "travis_retry pip install mock"
-  - "travis_retry pip install --pre -e ."
+- travis_retry pip install --pre -e .[testing,webassets]
 
-script: "nosetests -v --with-coverage --cover-package=dukpy --cover-erase"
+script:
+- nosetests -v --with-coverage --cover-package=dukpy --cover-erase
 
 after_success:
-  coveralls
+- coveralls
+
+stages:
+- test
+- name: deploy
+  if: tag IS present
+
+jobs:
+  include:
+  - stage: deploy
+    python: *mainstream_python
+    install: skip
+    script: skip
+    deploy:
+      provider: pypi
+      on:
+        tags: true
+        all_branches: true
+      user: amol-
+      # just source dist, because binary wheel requires extra work
+      distributions: sdist
+      password:
+        secure: YOUR_ENCRYPTED_PASSWORD_HERE
```

Comment:
@amol- 

Password placeholder is here. Feel free to **push into my branch** (GitHub has feature to allow such write access).

Reply:
Done, but note that I suppose it won't work until the branch is merged into mine. As least if I properly understood how encrypted password work.